demo/card/x-cards-list.html

```diff
@@ -46,29 +46,31 @@
 
     properties: {
       animationConfig: {
-        value: function() {
-          return {
-            'entry': [{
-              name: 'reverse-ripple-animation',
-              id: 'reverse-ripple',
-              toPage: this
-            }],
-
-            'exit': [{
-              name: 'fade-out-animation',
-              node: this.$.container,
-              timing: {
-                delay: 150,
-                duration: 0
-              }
-            }, {
-              name: 'ripple-animation',
-              id: 'ripple',
-              fromPage: this
-            }]
-          };
-        }
+        type: Object
       }
+    },
+
+    attached: function() {
+      this.animationConfig = {
```

Comment:
this is different from 1.x, as at every `attached` you'd override `this.animationConfig`
WDYT of just having a check before this line like `if (this.animationConfig) return;`?